src/dom-observer.js

```diff
@@ -2,9 +2,20 @@ define([
   './node'
 ], function (nodeHelpers) {
 
-  var MutationObserver = window.MutationObserver ||
-    window.WebKitMutationObserver ||
-    window.MozMutationObserver;
+  var MutationObserver;
+  // This enables server side rendering
+  if (typeof window !== 'undefined') {
```

Comment:
Turn this into a positive statement, e.g.  ===  undefined and consider extracting it into a function.